lib/index.ts

```diff
@@ -11,12 +12,12 @@ export {
   inspect,
 };
 
-function inspect(root: string, targetFile?: string, options?: any) {
```

Comment:
@orkamara I can't remember why we remove `?` from the argument `targetFile`, isn't it an optional argument that is provided only with the `--file` option in the cli?